tgui/packages/tgui-panel/chat/constants.ts

```diff
@@ -108,6 +109,12 @@ export const MESSAGE_TYPES = [
     description: 'Urist McTraitor has stabbed you with a knife!',
     selector: '.danger',
   },
+  {
+    type: MESSAGE_TYPE_SUBTLE, // NOVA EDIT
+    name: 'Subtle',
+    description: 'Subtle and Subtler actions.',
+    selector: '.subtle, .subtler',
+  },
```

Comment:
```suggestion
  { // NOVA EDIT ADDITION START
    type: MESSAGE_TYPE_SUBTLE,
    name: 'Subtle',
    description: 'Subtle and Subtler actions.',
    selector: '.subtle, .subtler',
  }, // NOVA EDIT ADDITION END
```
The correct way to format these edits (while you're in this file, would you mind fixing it for the mentorchat one too?)